feat: Adds new tokens to `Skeleton` (#171)
vtex-sites/nextjs.store: src/components/skeletons/Skeletons.stories.mdx

import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs'

import { SectionList, SectionItem } from 'src/../.storybook/components'

import Skeleton from './Skeleton'
import FilterSkeleton from './FilterSkeleton'
import ProductCardSkeleton from './ProductCardSkeleton'

import Icon from 'src/components/ui/Icon'
import ButtonLink from 'src/components/ui/Icon'

<Meta title="Features/Skeletons/Overview" />

<header>

# Skeletons

> This feature improves the store's experience and perceived performance through loading placeholders.

Reviewer: Nice question! I also wonder about it.

> This feature improves the store's experience and perceived performance through loading placeholders.

Author: I discussed this with @renatamottam, and it makes sense to be a feature for her, also I agree that it is ok so then we group all skeletons

</header>

## Overview

> <code>Skeletons</code> are like an element's or section's wireframe. The wireframe
> is a placeholder for these elements/sections until the page finally loads.

Reviewer 2: ```suggestion
<code>Skeletons</code> are like an element's or section's wireframe. It's a placeholder that simulates the layout of these elements/sections while data is being loaded.
```

Instead of showing a blank page, displaying <code>Skeletons</code> while loading content makes the user feel like the store is more responsive and faster.

Also by using <code>Skeletons</code> the store will be less likely to experience [Cumulative Layout Shilf (or CLS)](https://web.dev/i18n/en/cls/).

---

## Components

> <SectionList classes="sbdocs-skeletons">

Reviewer: It's up to you but what do you think about the grid version?
```suggestion
<SectionList classes="sbdocs-skeletons" grid="grid">
```
<img width="1338" alt="Screen Shot 2022-07-19 at 20 23 59" src="https://user-images.githubusercontent.com/11325562/179864618-1381bb4b-110b-4455-b45b-0c8377b7217d.png">

Reviewer 2: Agreed! 👍
